components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -210,7 +215,8 @@ func main() {
 
 	// define steps
 	provisioningInit := provisioning.NewInitialisationStep(db.Operations(), db.Instances(),
```

Comment:
Please move this parameter initialization `provisioning.NewConfigMapKymaVersionConfigurator(ctx, cli, cfg.VersionConfig.Namespace, cfg.VersionConfig.Name, logs)` to a separate variable. It will improve a bit the readability of this yet over parametrized function.

I'd go with something like this
```go
	// define steps
	kymaVerConfigurator := provisioning.NewConfigMapKymaVersionConfigurator(
		ctx, cli, cfg.VersionConfig.Namespace, cfg.VersionConfig.Name, logs)
	provisioningInit := provisioning.NewInitialisationStep(db.Operations(), 
		db.Instances(), provisionerClient, directorClient, inputFactory, 
		externalEvalCreator, iasTypeSetter, cfg.Provisioning.Timeout, 
		kymaVerConfigurator)
```

Reply:
ok, done